docs/learn/sidebars.ts

```diff
@@ -39,6 +39,141 @@ module.exports = {
         'chrysalis/tips',
       ],
     },
+    {
+      type: 'category',
+      label: 'Smart Contracts',
+      collapsed: true,
+      items: [
+        {
+          type: 'category',
+          label: 'Core Concepts',
+          items: [
```

Comment:
I think we should remove this category and just move all the categories/docs on level up, unless you plan to add the introduction and the core contracts categories as per the miro board.

Reply:
I thought we might add some other sections here. Core concepts is kind of an "advanced" topic. We will see in our final review